QRSDetectorOffline.py

```diff
@@ -58,30 +58,31 @@ class QRSDetectorOffline(object):
     SOFTWARE.
     """
 
-    def __init__(self, ecg_data_path, verbose=True, log_data=False, plot_data=False, show_plot=False):
+    def __init__(self, ecg_data_path, verbose=True, log_data=False, plot_data=False, show_plot=False,signal_frequency=250):
```

Comment:
That - I mean here adding signal_frequency parameter - is a great idea. The problem with this PR change is that you did your changes only in Offline detector leaving Online version interface without changes which will desynchronise them for future usage.

As mentioned in PR comment - we cannot accept these changes right now but will remember about these suggestions in the future.